views/js/pciCreator/dev/audioRecordingInteraction/runtime/scss/audioRecordingInteraction.scss

```diff
@@ -158,6 +200,14 @@ $playback: whiten($info, .3);
       height: 10px;
       border-radius: 5px;
 
```

Comment:
Would you please check this?
I think it is working fine without this code. The `dir` param is already changing the progressBar direction.
With CSS code
![image](https://user-images.githubusercontent.com/11692751/141097470-df464dbc-ec8e-47cf-8c4d-6dfde812246d.png)
Without it
![image](https://user-images.githubusercontent.com/11692751/141097558-ef3c77fe-fe47-47ac-931c-432a9ba33b79.png)